docs/book/src/component-config-tutorial/testdata/project/config/rbac/kustomization.yaml

```diff
@@ -16,3 +16,9 @@ resources:
 - auth_proxy_role.yaml
```

Comment:
@joelanford 

You request this one. What do you think?
Is that addressing your need?